docs/developer-docs/integrations/index.md

```diff
@@ -1,62 +1,35 @@
-# Functionality integrations
+# Advanced features
 
 ## Overview
 
 While previous sections guide you to start building canisters on the IC, here you can see how to integrate various (sometimes advanced) extra functionality to your dapp. From identity to ledger integrations, to integrating with other blockchains, to communicating from the IC to the outside world, and even decentralizing control of your dapp, it's all here.
 
-## Bitcoin Integration
-Integrate directly with the Bitcoin network allowing canisters on the IC to receive, hold, and send Bitcoin, all directly with transactions on the Bitcoin network. Canisters can act exactly like regular users holding bitcoin on the Bitcoin network.
-
-- [Bitcoin Integration](./bitcoin/index.md).
-- [How it works](./bitcoin/bitcoin-how-it-works.md).
-- [Local development](./bitcoin/local-development.md) contains a tutorial showing how to experiment locally.
-- [Chain-key Bitcoin](./bitcoin/ckbtc.md) provides an overview of chain-key Bitcoin (ckBTC).
-
 ## HTTPS Outcalls
```

Comment:
are we missing a section about composite queries in here as well?

Reply:
Yes, I have resolved this by adding a section on Composite Queries.